_contrib/commits.md

```diff
@@ -1,4 +1,7 @@
-# Git Commit Guidelines
+---
+title: Git Commit Guidelines
+layout: guide
+---
```

Comment:
I updated the CI command to convert the titles to frontmatter, as well as replace '_' with '-'. The layout is tbd, not sure if I can reuse it yet or need a new one.

Reply:
I did end up reusing the layout and I think it looks okay. The only thing missing now is fixing all the links, which could be a bit tricky since some of them use local urls and some of them use github urls. A PR to change all the links to local in the hyper repo may be needed, and then we can can just lowercase them in the CI workflow.

And also switching from the "commit to master" strategy to using PRs, but that should be easy.

Reply:
Local links makes sense to me.